google/resource_compute_instance_template.go

```diff
@@ -420,6 +420,48 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 	}
 }
 
+func resourceComputeInstanceTemplateCustomizeDiff(diff *schema.ResourceDiff, meta interface{}) error {
```

Comment:
nit: I'd call this `resourceComputeInstanceTemplateSourceImageCustomizeDiff`, since presumably if we added more fields to customize we would want to do a customdiff.All to group them.

(also lol, every so often i think that maybe our resources should be like, objects, so we don't have to have these ridiculously long globally-unique function names)

Reply:
Fair! I've been doing massive monolithic CustomizeDiff functions, but I think customdiff.All--which I just learned about!--is a better approach.

And yeah, long names are amazing. -_-